README.md

```diff
@@ -57,7 +57,9 @@ To run the OAuth example, you will need to have your Kafka cluster configured wi
 ## Configuration
 
 Although this Hello World is simple example it is fully configurable.
-Below are listed and described environmental variables.
+Below are listed and described environmental variables that are NOT Kafka-specific but must be used. 
+These environmental variables should be prefixed with `STRIMZI_` using `_` instead of `.` (eg. STRIMZI_TOPIC). 
+Any Kafka configuration option can be given as an environmental variable and should be prefixed with `KAFKA_` using `_` instead of `.` (eg. bootstrap.servers -> KAFKA_BOOTSTRAP_SERVERS).
```

Comment:
So, does this currently apply to consumers only? It is fine to do it step by step. But you should make it clear here in the README. Right now it sounds like this applies for producers and Streams as well.

Reply:
Refactored to specify Consumer only (until later updates)